routemaster/mixins/log.rb

```diff
@@ -8,15 +8,7 @@ module Log
       protected
 
       def _log
-        @@_logger ||= begin
-          file_path = ENV['ROUTEMASTER_LOG_FILE']
-          file = (file_path && File.exist?(file_path)) ? File.open(file_path, 'a') : $stderr
-          level = Logger.const_get(ENV.fetch('ROUTEMASTER_LOG_LEVEL', 'INFO'))
-          Logger.new(file).tap do |logger|
-            logger.level     = level
-            logger.formatter = method(:_formatter)
-          end
-        end
+        @@_logger ||= Services::Logger.new
```

Comment:
I know that this was already here in the previous version, but is this thread safe?
I guess it doesn't matter because it's not holding state, and the object is always configured in the same way.